core/src/test/resources/script/oracle-with-begin-end-blocks.sql

```diff
@@ -0,0 +1,40 @@
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+CREATE TABLE EXAMPLE
+(
+    ID            NUMBER PRIMARY KEY,
+    USERNAME      VARCHAR2(128) NOT NULL,
+    CREATED_AT    TIMESTAMP WITH TIME ZONE NOT NULL
+);
+
+CREATE SEQUENCE EXAMPLE_ID_SEQUENCE
+    START WITH 1
+    INCREMENT BY 1;
+
+CREATE OR REPLACE TRIGGER EXAMPLE_TRIGGER
+    BEFORE INSERT
+    ON EXAMPLE
+    FOR EACH ROW
+BEGIN
```

Comment:
Would it be worth testing that `BEGIN` and `END` are ok to include in comments?
like `-- a comment that includes BEGIN`

Reply:
I've amended the test but please be aware I am anything but an Antlr expert ;)